doc/changelog/03-notations/12324-master+nametab-method-remove.rst

```diff
@@ -0,0 +1,5 @@
+- **Added:**
+  command :cmd:`Enable Notation`
+  to enable or disable previously defined notations
+  (`#12324 <https://github.com/coq/coq/pull/12324>`_,
+  by Hugo Herbelin).
```

Comment:
Only Hugo, not Pierre, too?

Reply:
Yes, Pierre too. I pushed refined credits.

Reply:
I feel I only fixed a few final points and wrote the doc, but why not.
I also credited Jim's review which, as always, greatly improved the quality of the documentation.

Reply:
Now you need someone to review the code...